app/src/test/java/io/apicurio/registry/rest/v2/IdsResourceTest.java

```diff
@@ -124,7 +122,8 @@ public void testIdsAfterCreate() throws Exception {
                 .statusCode(200)
                 .body("count", equalTo(1))
                 .body("versions[0].contentId", notNullValue())
-                .body("versions[0].contentId", not(equalTo(0)))
+                //in memory storage starts generating ids from 1 , sql storage from 0
+                //.body("versions[0].contentId", not(equalTo(0)))
```

Comment:
This is why I added the "throwaway" artifact on line 47 - so that the contentId can't be 0 no matter what.  Did that not work for SQL??

Reply:
that's weird, I did not notice that detail. So regardless of what the counter strategy the storage uses you first create an dummy artifact just to ensure the contentId for the next artifacts is different than 0 ?
I have to dig deeper into this

Reply:
ok, this was my bad changing the test. That failure reflected a bug in the sql impl, which I already fixed

Reply:
btw, I reverted this change